File: sqlInsert.php
<?php
$servername = "localhost";
$username = "erizho21";
$password = "password";
$dbname = "school";

// Create connection
$conn = new mysqli($servername, $username, $password, $dbname);
// Check connection
if ($conn->connect_error) {
    die("Connection failed: " . $conn->connect_error);
}
// Insert Data
$sql = "INSERT INTO students (name, age, gradeLevel) VALUES ('Eric', '16', '11')";

if ($conn->query($sql) === TRUE) {
    echo "New record created successfully";
} else {
    echo "Error: " . $sql . "<br>" . $conn->error;
}

$conn->close();
?> 

File: sqlInsert.py
import MySQLdb
#Connecting to database with mysqldb
db = MySQLdb.connect(host="localhost", user="user", passwd="password", db="school")
db.autocommit(True)
#Creating cursor
cur = db.cursor(MySQLdb.cursors.DictCursor)
#Variables
nameIn = 'Eric'
ageIn = 16
gradeLevelIn = 11
#Sql command that inserts the variables via f string
sql = f"INSERT INTO students (name, age, gradeLevel) VALUES ('{nameIn}', '{ageIn}', '{gradeLevelIn}')"
#Cursor executing sql code
cur.execute(sql)
#Disconnecting cursor and database
cur.close()
db.close()
